src/api/requests/Websocket/Websocket.ts

```diff
@@ -1,45 +1,50 @@
-import {Centrifuge, MessageContext} from "centrifuge";
-import {BaseWebsocketModel, WebsocketTransaction} from '@/api/types/transaction'
+import { Centrifuge, MessageContext } from 'centrifuge'
+import { BaseWebsocketModel, WebsocketTransaction } from '@/api/types/transaction'
+import { toast } from 'react-toastify'
 
 export const SetupWebsocket = (url: string) => {
-    let retries = 0;
-    const centrifuge = new Centrifuge(url, {websocket: WebSocket});
-    centrifuge.connect();
-
-    centrifuge.on('message', function (ctx: MessageContext) {
-        const res = parseResponse(ctx)
-
-        if (typeof res == "object" && res != undefined){
-            switch (res.eventType) {
-                case "create_transaction":
-                    // TODO: create notification
-                    break;
-            }
-        }
-    });
-
-    centrifuge.on('error', function () {
-        retries++
-        if (retries >= 3){
-            centrifuge.disconnect()
-        }
-    })
-
-    const channel = "bux-wallet";
-    const sub = centrifuge.newSubscription(channel);
-    sub.subscribe();
+  let retries = 0
+  const centrifuge = new Centrifuge(url, { websocket: WebSocket })
+  centrifuge.connect()
+
+  centrifuge.on('message', function (ctx: MessageContext) {
+    const res = parseResponse(ctx)
+
+    if (typeof res == 'object' && res != undefined) {
+      switch (res.eventType) {
+        case 'create_transaction':
+          if (res.status === 'success') {
+            toast.success('Transaction successfully sent')
+          } else if (res.status === 'error') {
+            toast.error('Error while sending transaction')
```

Comment:
Is there any detailed information about error in the "event?
If so, even though it could be "low level error", it could be useful to display it, for those who will start this wallet to play around with bux-server.

Reply:
Changed text from event